package.json

```diff
@@ -74,6 +74,7 @@
     "matrix-js-sdk": "0.8.5",
     "optimist": "^0.6.1",
     "prop-types": "^15.5.8",
+    "qs": "^6.5.1",
```

Comment:
I forgot about this. We already use querystring in riot-web. Let's use that? (See url_utils.js in riot-web)

Reply:
Ok, I've updated again to reflect (I had querystring earlier in the PR, but reverted to qs as it is already being installed as a dep of other imports in react-sdk).

It's frustrating that riot-web's package.json doesn't list querystring as a dep., which would have made our use / choice of it more obvious.